functions/auth.js

```diff
@@ -12,7 +12,7 @@ const auth = (req, res, next) => {
         .json(genereteJSON("error", 401, "error message", "Not authorized"));
     }
     req.user = user;
-    next();
+    next(error);
```

Comment:
Ten error nie zostanie wyslany, gdyz warunek masz wczesniej, ze jesli jest error to wykona sie linia 10.

Reply:
Czyli poprawką powinno być, usunięcie next czy pozostawienie next() bez argumentu?

Reply:
Pozostawienie next bez argumentu